x-pack/plugins/uptime/e2e/journeys/utils.ts

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Can we add tests for this file please?

Reply:
this is already a test file util , it will become an inception 😄